### PR TITLE
i3altlayout: init at 0.3

### PR DESCRIPTION
**Pull request by magnetophon**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/window-managers/i3/altlayout.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+python3Packages.buildPythonApplication rec {
+  pname = "i3altlayout";
+  version = "0.3";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "1h0phf3s6ljffxw0bs73k041wildaz01h37iv5mxhami41wrh4qf";
+  };
+
+  nativeBuildInputs =  with python3Packages; [ enum-compat i3ipc docopt ];
```
Comment (symphorien) [inline pkgs/applications/window-managers/i3/altlayout.nix:12]: this line looks unnecessary (it builds without)

File: pkgs/applications/window-managers/i3/altlayout.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  pythonPath = with python3Packages; [ enum-compat i3ipc docopt ];
+
+  meta = with lib; {
+    maintainers = with maintainers; [ magnetophon ];
+    description = "helps you handle more efficiently your screen real estate in i3wm by auto-splitting windows on their longest side.";
```
Comment (symphorien) [inline pkgs/applications/window-managers/i3/altlayout.nix:17]:
```suggestion
    description = "Helps you handle more efficiently your screen real estate in i3wm by auto-splitting windows on their longest side";
```

See https://nixos.org/manual/nixpkgs/unstable/#sec-standard-meta-attributes

**Review by symphorien — CHANGES_REQUESTED** [review]
Looks good apart from a few nitpicks.

**Comment by magnetophon** [comment]
@symphorien Thanks for the feedback.
All done!

**Pull request merged**